Mianta/06_challenges/stocks_challenge.py

```diff
@@ -0,0 +1,60 @@
+from http import client
+
+
+print("Challenge 3.2: Playing with the stock market")
+
+print()
+
+# Creating variables to store the current (approximate) market price of these 5 companies - Amazon, Apple, Facebook, Google and Microsoft.
+
+amzn = 3000
+apple = 100
+fb = 250
+google = 1400
+msft = 200
+
+print("Challenge 3.2.1: Taking user input")
+# asked the client their name and saved it to the variable.
+client_name = input ('Please enter your name...-')
+if client_name.isdigit() == False:
+     print('Please re-enter your name-')
+client_name = input ('Please enter your name-') #this ensures numerical input is not input but instead alphabets 
+
+#asked client what their savings account amount and saved it to another variable.
+client_savings = int(input(f'Hello,{client_name} and Thank you! How much is in your savings account?'))
+
+#Wrote code to asked the client the stock they are interested in and saved it to another variable, as shown below.
+stock = input("Awesome, thank you! Which stock are you interested in? Please type 'amzn' for Amazon, 'appl' for Apple, 'fb' for Facebook, 'goog' for Google and 'msft' for Microsoft.")
+print('Uh-Oh! Please re-type your preferred stock only alphabet letters.')
```

Comment:
Here, you should use an “if else” statement to check whether the user input is in the correct format. Right now, your code will print out 'Uh-Oh! Please re-type your preferred stock only alphabet letters.' no matter what the user input is.

Reply:
I am still getting the error message in the terminal when I run the code. I am not sure how to fix this

Reply:
if stock not in ['amzn', 'appl', 'fb','goog', 'msft']:
       print('Uh-Oh! Please re-type your preferred stock only alphabet letters.')
       stock = input("")

Reply:
I understand why now because of the line above that says what the stock is.